json_schemas/test_story.schema.yaml

```diff
@@ -55,7 +55,7 @@ definitions:
       - $ref: '#/definitions/ChapterRequest'
       - type: object
         properties:
-          additional_success_status:
+          additional_success_statuses:
```

Comment:
I would rename this to `status` and override the default value of `200` or `201` so that there's no hidden functionality.